test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be ideally ignored by git. As such, it should not be committed to your project. Kindly re-add the test file in `.gitignore`, and make sure this does not show, as that's the essence of the `.gitignore` file